### PR TITLE
Jesse/pizza

### PR DESCRIPTION
**Pull request by pyromelonz**
rythm manager with swing, graphically represented like pizza slices. Works with beam in manapool scene

### CODE REVIEW

File: levelslice.gd
Comment (davidelrosas) [inline levelslice.gd:1]: this not ready

File: prefabs/Visualpizza.gd
Comment (davidelrosas) [inline prefabs/Visualpizza.gd:1]: whar is this why _ready and _process empty

File: prefabs/range_detector.gd
Comment (davidelrosas) [inline prefabs/range_detector.gd:1]: empty

File: prefabs/range_detector.tscn
Comment (davidelrosas) [inline prefabs/range_detector.tscn:1]: what is this

File: scripts/Pizza.gd
Comment (davidelrosas) [inline scripts/Pizza.gd:1]: can we please have more descriptive variable names

Comment (davidelrosas) [reply scripts/Pizza.gd:1]: its only peegle sorry for overreact

**Review by davidelrosas — CHANGES_REQUESTED** [review]

File: scripts/Pizza.gd
Comment (pyromelonz) [reply scripts/Pizza.gd:1]: suck my peegle

**Review by davidelrosas — APPROVED** [review]

**Pull request merged**